frontend/src/components/candidates/candidateInterviewsModal.js

```diff
@@ -53,10 +53,16 @@ class CandidateInterviewsModal extends Component {
   }
 
   render() {
+    const closeBtn = (
+      <button className="close" onClick={this.props.exitModal}>
+        &times;
+      </button>
```

Comment:
Nit: can we move this out to a property of the class rather than inside `render`?